Removed platform-specific code for painting DropShadow

spireken opened the pull request:
I updated the Login Window, Security Input, and the Time and Sales Window.

The Security Input window uses the Drop Shadow for the dropdown list, and the Time and Sales Window uses the Drop Shadow for it's context menu.

spireken commented:
Muscle memory strikes again.

I just updated the Login Window build. The Time and Sales Window build demonstrates the security input and context menus uses of the drop shadow.

kranar commented:
Ask Darryl to look it over, if he says it's good then I'll approve it.

On Mon, Nov 19, 2018 at 5:41 PM spireken <notifications@github.com> wrote:

> Muscle memory strikes again.
>
> I just updated the Login Window build. The Time and Sales Window build
> demonstrates the security input and context menus uses of the drop shadow.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/eidolonsystems/nexus/pull/265#issuecomment-440068614>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/APzFGUEMk68L_Q_EvQH2lEqvaYEF3AfIks5uwzOdgaJpZM4Ypcz9>
> .
>


-- 
Kamal Mansouri
Computer Programmer
+1 647-648-0498

spireken commented:
Okay, I still have to fix that issue, though. I still don't know what's causing it because it only happens on the initial showing of the security input; if I type in 'ms' or type in 'm' then hit backspace, it displays properly.

spireken commented:
Other than that, Darryl says it's okay.

spireken commented:
How would you recommend I implement this? Should DropShadow have a static member that keeps a list of created drop shadows?

spireken commented:
Nevermind, I can just compare HWND's, like in the Window class.

spireken commented:
I updated the Time and Sales, Book View, and Login Window builds.

kranar commented:
Did Darryl give the ok?

On Mon, Jan 7, 2019 at 12:39 PM spireken <notifications@github.com> wrote:

> I updated the Time and Sales, Book View, and Login Window builds.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/eidolonsystems/nexus/pull/265#issuecomment-452017181>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/APzFGcAswHqEpyZF26_fr1iTodViv8Dkks5vA4Y6gaJpZM4Ypcz9>
> .
>


-- 
Kamal Mansouri
Computer Programmer
+1 647-648-0498

spireken commented:
Do I merge spire2018 into this branch to resolve these conflicts?

kranar commented:
Merging won't solve a conflict, you'll need to open up an editor and
resolve them individually.

On Mon, Jan 7, 2019 at 3:31 PM spireken <notifications@github.com> wrote:

> Do I merge spire2018 into this branch to resolve these conflicts?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/eidolonsystems/nexus/pull/265#issuecomment-452071482>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/APzFGS8G3KMaVjFJKTKwQOHAQRD9V0gCks5vA66KgaJpZM4Ypcz9>
> .
>


-- 
Kamal Mansouri
Computer Programmer
+1 647-648-0498

spireken commented:
It's mine to fix, though, was what I was asking.

kranar commented:
Yes, all conflicts need to be resolved before your branch gets merged.

On Mon, Jan 7, 2019 at 3:34 PM spireken <notifications@github.com> wrote:

> It's mine to fix, though, was what I was asking.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/eidolonsystems/nexus/pull/265#issuecomment-452072448>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/APzFGdIDaocKTEEuFhVgldWUsm_HJFv_ks5vA69WgaJpZM4Ypcz9>
> .
>


-- 
Kamal Mansouri
Computer Programmer
+1 647-648-0498

kranar commented on Applications/spire/source/login/login_window.cpp:
```diff
@@ -18,6 +18,7 @@ LoginWindow::LoginWindow(const std::string& version, QWidget* parent)
   setWindowIcon(QIcon(":icons/spire-icon-256x256.png"));
   setFixedSize(scale(396, 358));
   m_shadow = std::make_unique<DropShadow>(this);
+  m_shadow->setWindowFlag(Qt::WindowStaysOnTopHint, false);
```
What happens if this is left ```true```?

spireken replied:
Nothing changes; I removed it.

kranar commented on Applications/spire/source/ui/drop_shadow.cpp:
```diff
@@ -1,8 +1,10 @@
 #include <QLinearGradient>
 #include <QPainter>
 #include <QRadialGradient>
+#include "qt_windows.h"
```
Where is this header found?

spireken replied:
What's wrong with it? Should it be <qt_windows.h>?

kranar replied:
Yes.

spireken commented:
I updated the Login Window build on Dropbox. For some reason on my machine it's not updating the timestamp for the executable, but I deleted the old one before rebuilding the project and the one I pasted into Dropbox is the one the build produced.
